cirq/contrib/qasm_import/_parser.py

```diff
@@ -37,13 +39,28 @@ def __init__(self, supported_format: bool, qelib1_include: bool,
 
 
 class QasmGateStatement:
+    """Specifies how to convert a call to an OpenQASM gate
+        to a list of `cirq.GateOperation`s.
+
+        Has the responsibility to validate the arguments
+        and parameters of the call and to generate a list of corresponding
+        `cirq.GateOperation`s in the `on` method.
+        """
 
     def __init__(self, qasm_gate: str, cirq_gate: cirq.Gate, num_args: int):
         self.qasm_gate = qasm_gate
         self.cirq_gate = cirq_gate
         self.num_args = num_args
+        """Initializes a Qasm gate statement.
```

Comment:
Docstring goes after def line

Reply:
god - that's why I shouldn't commit/push while on a meeting :)